compilation fails with bazel in resty 2.8

while it works fine with `go build`, with `bazel build` it fails.

```
bazelisk build //...
INFO: Invocation ID: 93e6226f-ae83-49ae-9202-4a7030c7fd62
INFO: Analyzed 116 targets (1 packages loaded, 13 targets configured).
INFO: Found 116 targets...
ERROR: /home/pdeva/.cache/bazel/_bazel_pdeva/b1a4ac6988a94eb0ad1222a792d4139b/external/gazelle~0.33.0~go_deps~com_github_go_resty_resty_v2/BUILD.bazel:8:11: GoCompilePkg external/gazelle~0.33.0~go_deps~com_github_go_resty_resty_v2/resty.a failed: (Exit 1): builder failed: error executing command (from target @gazelle~0.33.0~go_deps~com_github_go_resty_resty_v2//:resty) bazel-out/aarch64-opt-exec-2B5CBBC6/bin/external/rules_go~0.41.0~go_sdk~astradot_monorepo__download_0/builder_reset/builder compilepkg -sdk external/rules_go~0.41.0~go_sdk~astradot_monorepo__download_0 ... (remaining 46 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
external/gazelle~0.33.0~go_deps~com_github_go_resty_resty_v2/client.go:420:29: undefined: digestTransport
external/gazelle~0.33.0~go_deps~com_github_go_resty_resty_v2/client.go:421:23: undefined: digestCredentials
external/gazelle~0.33.0~go_deps~com_github_go_resty_resty_v2/request.go:522:29: undefined: digestTransport
external/gazelle~0.33.0~go_deps~com_github_go_resty_resty_v2/request.go:523:23: undefined: digestCredentials
external/gazelle~0.33.0~go_deps~com_github_go_resty_resty_v2/transport.go:28:26: undefined: transportDialContext
compilepkg: error running subcommand external/rules_go~0.41.0~go_sdk~astradot_monorepo__download_0/pkg/tool/linux_arm64/compile: exit status 2
INFO: Elapsed time: 7.155s, Critical Path: 0.08s
INFO: 2 processes: 2 internal.
FAILED: Build did NOT complete successfully
make: *** [Makefile:56: build] Error 1
```

seems you havent put `digest.go` in your `BUILD.bazel` file

--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -9,6 +9,7 @@ go_library(
     name = "resty",
     srcs = [
         "client.go",
+        "digest.go",
         "middleware.go",
         "redirect.go",
         "request.go",
@@ -16,6 +17,8 @@ go_library(
         "resty.go",
         "retry.go",
         "trace.go",
+        "transport_js.go",
+        "transport_other.go",
         "transport.go",
         "transport112.go",
         "util.go",

